content/en/events/guides/migrating_to_new_events_features.md

```diff
@@ -6,35 +6,59 @@ aliases:
+  Use this guide to migrate from the legacy Events Stream to the Events Explorer. The Events Explorer is being automatically rolled out to all customers in Q1 and Q2 2022. If you see the <strong>Explorer</strong> option under <strong>Events</strong> in the left navigation bar, you have the new features. For legacy events information, see the <a href="/events/stream/">Event Stream</a>.</div>
 
-The Event Stream is one of Datadog’s oldest features. The introduction of the Events Explorer and its associated changes means that Event query syntax, explorer and analytics views, processing pipelines, and other features work the same way they do for other Datadog products such as Logs. See [Events Explorer][1] for more details. 
+The Events Explorer replaces the Event Stream, one of Datadog's oldest features. The new Event query syntax requires migration of Event monitors, dashboard widgets, and any other features that query Events. 
+
+The introduction of the Events Explorer and its associated changes means that Event features work similarly to other Datadog products. Event query syntax, explorer and analytics views, processing pipelines and other features look like their counterparts in Logs or APM Traces. See [Events Explorer][1] for more details.
 
 ## What this means for your organization
 
 * New events (after migration) are visible only in the [Events Explorer][2] or dashboard widgets and event overlays.
 
 * Older events (before migration) continue to be visible in the [Event Stream][2] for the remainder of their retention period (13 months).
 
-This guide describes how to migrate your organization’s event monitors and dashboard widgets so that you benefit from the new Events features, and so that you're ready when the legacy event stream and storage are sunset. It also provides [a detailed description of what's changed](#details-of-the-changes).
+This guide describes how to migrate your organization’s event monitors and dashboard widgets so that you benefit from the new Events features. Following this guide ensures that you are ready when Datadog retires the legacy event stream and storage. This guide also provides [a detailed description of what's changed](#details-of-the-changes).
 
 ## Migration process
 
-1. **Request early access.** Events Explorer is in private beta. [Contact support][3] to request access for your organization. The support team sends you a notification when Events Explorer is enabled for your organization. You can access the new Explorer and other features through the Events menu in the main navigation, alongside the existing Stream.
+### Datadog migrates data stored in Datadog servers
+<div class="alert alert-info">
+You do not need to take any action for configuration stored directly in Datadog.
```

Comment:
You do not need to take any action for any dashboards or monitors created via Datadog.